src/components/NavigationBar/index.js

```diff
@@ -3,8 +3,11 @@ import classNames from 'classnames'
 import style from './style.css'
 import {preventDefaultOnClick} from '../utils'
 
-const NavigationBar = ({back, i18n, disabled, isFullScreen, className}) =>
-  <div className={classNames(className, style.navigation, {[style.fullScreenNav]: isFullScreen})}>
+const NavigationBar = ({back, i18n, disabled, isFullScreen, isLiveness, className}) =>
+  <div className={classNames(className, style.navigation, {
+    [style.fullScreenNav]: isFullScreen,
+    [style.livenessNav]: isLiveness,
```

Comment:
hmm, can the liveness screen just use the `isFullScreen` considering it has the same style? Looking at the css it seems difference is on desktop and mobile?

The problem I have more is that the navigation bar is aware of something like `Liveness`. One should be able to control different modes of the navigation bar, but I don't think the navigation bar should be aware of different components.

Reply:
Couldn't agree more. 

There is a problem with how it currently works, however.  Children components can change the fullscreen state via `useFullScreen`. 

Currently, when fullscreen is enabled, on mobile components will look fullscreen, but on desktop they will look the same.  

My initial work on this PR removed the exception of fullscreen not working on desktop, as Liveness does have to look fullscreen on desktop according to designs. 

The problem is `Photo` still doesn't have too look fullscreen on desktop. So the exception of desktop not showing as fullscreen needs to remain, except for Liveness where we do have to show in fullscreen mode on desktop too.

So now we not only need to set `fullscreen` to `true`, the knowledge of whether is `Liveness` or not is relevant for the whole app, including `Overlay`, `NavigationBar` and a few other components higher up in the component tree.

Reply:
@rfreitas, maybe we can rename `isLiveness` to `isFullscreenDesktop`?